README.md

```diff
@@ -26,4 +26,39 @@ Push the branch:
 ```
 git push origin <your-branch-name>
 ```
-Create a pull request and wait for review.
\ No newline at end of file
+Create a pull request and wait for review.
+
+## Faster `Braid` implementation
+
+If you want to use the faster `Braid` implementation in `braid_vec.py`, you need 
+to compile the C++ modules.
+
+> ⚠️ Warning: although the APIs are very similar, some features, such as negative 
+indexing is not supported in the faster implementations.
```

Comment:
All lines of blockquotes should have `>` prefixed.

Reply:
Done